recipe/meta.yaml

```diff
@@ -50,6 +50,8 @@ about:
   license_family: APACHE
   license_file: LICENSE
   summary: A library to access the metadata for a Python package
+  dev_url: https://github.com/python/importlib_metadata
+  doc_url: https://github.com/python/importlib_metadata/blob/main/README.rst
```

Comment:
when I take a look to pypi, there the doc_url is given as https://importlib_metadata.readthedocs.io/ ... so why you point to a specific blob there?

Reply:
The doc_url has been modified:
https://github.com/AnacondaRecipes/importlib_metadata-feedstock/blob/73713d319acc1d23645f3637289e0d035a8ed9ee/recipe/meta.yaml#L54